### PR TITLE
array/split

### PR DESCRIPTION
**Pull request by osharej**

### CODE REVIEW

File: Jeff_pet_codes/ArrayThings/src/Student.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        studentScores.add(55);
+        studentScores.add(87);
+        studentScores.add(78);
+
+        return studentScores;
+
+
+    }
+
+    public void assignStudentScore() {
```
Comment (edwinchukus) [inline Jeff_pet_codes/ArrayThings/src/Student.java]: The method name does not reflect what the code content is doing. Method name says, assign student score and this is the only thing it needs to do and not going on to print things out. Take note.

File: Jeff_pet_codes/ArrayThings/src/Student.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        scores.set(4, 100);
+        for (int studentScore:scores){
+            System.out.println(studentScore);
+        }
+            System.out.println("size = " + scores.size());
+
+    }
+
+    public void swapStudentScore() {
+
```
Comment (edwinchukus) [inline Jeff_pet_codes/ArrayThings/src/Student.java]: Again, the same correction. Just swap it and not printing out. If you need to print out, just create method like "swapScoreAndDisplayResult()"

File: Jeff_pet_codes/ArrayThings/src/StudentGrade.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by Jeffries Pc on 24/07/2017.
+ */
+public class StudentGrade {
+
+
+    public String[] splitStudentGrade (){
+    String studentGrades = ("Pass,Fail,Pass,Resit,Pending");
```
Comment (edwinchukus) [inline Jeff_pet_codes/ArrayThings/src/StudentGrade.java]: This is good but I don't understand why you have opening and closing brackets while declaring string studentGrades?

File: Jeff_pet_codes/ArrayThings/src/StudentRunner.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    public static void main(String[] args) {
+
+        Student student = new Student();
+        student.assignStudentScore();
+        student.swapStudentScore();
+        StudentGrade freshStudent = new StudentGrade ();
+        freshStudent.splitStudentGrade();
+
+    }
+
```
Comment (edwinchukus) [inline Jeff_pet_codes/ArrayThings/src/StudentRunner.java:12]: Overall, I am incredibly happy with your performance.

**Review by edwinchukus — CHANGES_REQUESTED** [review]
Hi Jeff, I am please with your progress but it seems that you were really pre-occupied with something else lately. It shows in the level of your interest in Java programmaing. Kindly bounce back faster please. I have listed minor issues with your codes with are trivials but I need you to correct them and we can merge the code. Thank you

File: Jeff_pet_codes/ArrayThings/src/StudentGrade.java (hunk 3 above)
Comment (osharej) [reply Jeff_pet_codes/ArrayThings/src/StudentGrade.java]: _I wonder how that bracket came into the string declaration LOL.

File: Jeff_pet_codes/ArrayThings/src/StudentRunner.java (hunk 4 above)
Comment (osharej) [reply Jeff_pet_codes/ArrayThings/src/StudentRunner.java:12]: changes will follow shortly

Comment (tccanini) [reply Jeff_pet_codes/ArrayThings/src/StudentRunner.java:12]: I think this code has been reviewed and updated as advised

Comment (tccanini) [reply Jeff_pet_codes/ArrayThings/src/StudentRunner.java:12]: Kindly approve review

File: Jeff_pet_codes/ArrayThings/src/Student.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        studentScores.add(75);
+        studentScores.add(99);
+        studentScores.add(55);
+        studentScores.add(55);
+        studentScores.add(87);
+        studentScores.add(78);
+
+        return studentScores;
+
+
```
Comment (tccanini) [inline Jeff_pet_codes/ArrayThings/src/Student.java:23]: This line looks good to me

**Review by edwinchukus — APPROVED** [review]
This is fine with me now.

**Pull request merged**